draft-ietf-core-senml.md

```diff
@@ -518,14 +519,27 @@ by the application context.
 
 ### Time
 
-If either the Base Time or Time value is missing, the missing
-field is considered to have a value of zero. The Base Time and
-Time values are added together to get the time of measurement. A time
-of zero indicates that the sensor does not know the absolute time and
-the measurement was made roughly "now". A negative value is used to
-indicate seconds in the past from roughly "now". A positive value is
-used to indicate the number of seconds, excluding leap seconds, since
-the start of the year 1970 in UTC.
+If either the Base Time or Time value is missing, the missing field is
+considered to have a value of zero. The Base Time and Time values are
+added together to get the time of measurement. A time of zero
+indicates that the sensor does not know the absolute time and the
+measurement was made roughly "now". A negative value is used to
+indicate seconds in the past from roughly "now". 
+
+Positive values up to 2^28 indicate seconds in the future from "now".
+These can be used e.g., for actuation use when the desired change
+should happen in the future but the sender or the receiver does not
+have accurate time available.
+
+Values greater than 2^28 represent an absolute time relative to the
+Unix epoch (1970-01-01T00:00Z in UTC time) and the time is counted
+same way as the Portable Operating System Interface (POSIX) "seconds
+since the epoch" {{TIME_T}}.
```

Comment:
Maybe mention Time.at(2**28).utc => 1978-07-04 21:24:16 UTC as the minimum absolute time value that can be expressed this way.

Reply:
Fixed.